src/CarbonAware.WebApi/src/README.md

```diff
@@ -154,6 +159,57 @@ EG
 ```
 
 
+### GET forecasts/current
+
+This endpoint fetches the most recent forecast for all provided locations and calculates the optimal marginal carbon intensity windows (per the specified windowSize) for each, within the start and end time boundaries. 
+If no start or end time boundaries are provided, all forecasted data points are used. 
+
+The forecast data represents what the data source predicts future marginal carbon intesity values to be, not actual measured emissions data (as future values cannot be known).
+This endpoint is useful for determining if there is a more carbon-optimal time to use electicity predicted in the future.
+
+Parameters:
+1. location: This is a required parameter and is an array of the names of the data region for the configured Cloud provider.
+2. startTime: Start time boundary of forecasted data points. Ignores current forecast data points before this time. It defaults to the earliest time in the forecast data.
+3. endTime: End time boundary of forecasted data points. Ignores current forecast data points after this time. Defaults to the latest time in the forecast data.
+If time period is not provided, it retrieves all the data until the current time.
+4. windowSize: The estimated duration (in minutes) of the workload. Defaults to the duration of a single forecast data point.
+
+EG
+```
+https://<server_name>/forecasts/current?location=northeurope&startTime=2022-07-19 14:00&endTime=2022-07-20&windowSize=10
```

Comment:
This example does not work.  The time should be in this format:  2022-07-19T15:00:00.000Z.  It returns a 404 error.

Reply:
https://<server_name>/emissions/forecasts/current?location=eastus&windowSize=60&starttime=2022-07-15T03:38:00.000Z&endtime=2022-07-15T04:38:00.000Z

(works if you update the timestamp)